libsplinter/src/admin/rest_api/mod.rs

```diff
@@ -49,20 +36,19 @@ use self::actix::ws_register_type::make_application_handler_registration_route;
 ///
 /// * `rest-api-actix`
 impl RestResourceProvider for AdminService {
+    #[cfg(feature = "rest-api-actix")]
```

Comment:
I like the older pattern better (creating a vec, then filling it), because when we have more than one backend things are always stated in the positive, not "not x, y, z" as the empty implementation is now. A compromise would be to do:
```
#[cfg(feature = "rest-api-actix")]
{
    resources.push(actix::ws_register_type::make_application_handler_registration_route(self.commands()));
    ...
}
```

Thus reducing the cfg statements but leaving one resources() function.

Reply:
Updated